lib/origin_simulator/payload.ex

```diff
@@ -1,38 +1,38 @@
 defmodule OriginSimulator.Payload do
   use GenServer
 
-  alias OriginSimulator.{Body, Recipe, Size}
+  alias OriginSimulator.{Body, Recipe}
 
   ## Client API
 
   def start_link(opts) do
     GenServer.start_link(__MODULE__, opts, name: :payload)
   end
 
-  def fetch(server, %Recipe{origin: value}) when is_binary(value) do
-    GenServer.call(server, {:fetch, value})
+  def fetch(server, %Recipe{origin: value, route: route}) when is_binary(value) do
+    GenServer.call(server, {:fetch, value, route})
   end
 
-  def fetch(server, %Recipe{body: value}) when is_binary(value) do
-    GenServer.call(server, {:parse, value})
+  def fetch(server, %Recipe{body: value, route: route}) when is_binary(value) do
+    GenServer.call(server, {:parse, value, route})
   end
 
-  def fetch(server, %Recipe{random_content: value}) when is_binary(value) do
-    GenServer.call(server, {:generate, value})
+  def fetch(server, %Recipe{random_content: value, route: route}) when is_binary(value) do
+    GenServer.call(server, {:generate, value, route})
   end
 
-  def body(_server, status) do
+  def body(_server, status, route \\ Recipe.default_route()) do
     case status do
-      200 -> cache_lookup()
+      200 -> cache_lookup(route)
       404 -> {:ok, "Not found"}
       406 -> {:ok, "Recipe not set, please POST a recipe to /add_recipe"}
-      _   -> {:ok, "Error #{status}"}
+      _ -> {:ok, "Error #{status}"}
```

Comment:
If we run `mix format` does this get aligned with the other statements?

Reply:
Bizarrely, that was an outcome of running `mix format`!